src/client/src/App.vue

```diff
@@ -110,7 +150,7 @@
         if (recipients.value.length > 0) {
             sending.value = true;
             try {
-                await axios.post('/api/sendEmails', recipients.value);
+                await axios.post('/api/sendEmails', { participants: recipients.value, gameRules: gameRules.value });
```

Comment:
We should assign this to a `SendEmailPayload` to get the type checked. I had a bug initially where I named participants something different.

Reply:
Fixed